enos/modules/iam_setup/main.tf

```diff
@@ -9,9 +9,9 @@ locals {
 }
 
 resource "aws_iam_user" "boundary" {
-  name                 = "boundary-e2e-${var.test_id}"
-  tags                 = { boundary-demo = local.user_email }
-  permissions_boundary = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:policy/BoundaryDemoPermissionsBoundary"
+  name = "boundary-e2e-${var.test_id}"
+  #  tags                 = { boundary-demo = local.user_email }
```

Comment:
I wonder if we should delete these commented attributes rather than leave them as comments.